src/drem/transform/vo.py

```diff
@@ -129,60 +170,114 @@ def _convert_to_geodataframe(df: pd.DataFrame) -> gpd.GeoDataFrame:
     return gpd.GeoDataFrame(df, geometry=coordinates, crs="epsg:2157")
 
 
+@task
 def _set_coordinate_reference_system_to_lat_long(
     gdf: gpd.GeoDataFrame,
 ) -> gpd.GeoDataFrame:
 
     return gdf.to_crs("epsg:4326")
 
 
-@task
-def transform_vo(
-    vo_dirpath: Path, benchmarks: pd.DataFrame, unmatched_txtfile: Path,
-) -> gpd.GeoDataFrame:
+with Flow("Transform Raw VO") as flow:
+
     """Tidy Valuation Office dataset.
 
+     Args:
+        Task (prefect.Task): see https://docs.prefect.io/core/concepts/tasks.html
+        VisualizeMixin (object): Mixin to add flow visualization method
+
     By:
```

Comment:
Would you mind deleting this entire section from by: ... downwards?  No need - prefect flow viz better